.github/workflows/ci.yml

```
@@ -157,6 +157,47 @@ jobs:
     - run: test ${{ steps.setup3.outputs.mpi }} == intelmpi
     - run: command -v impi_info
 
+  container:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - debian
+          - ubuntu
+    container: ${{ matrix.image }}
+    steps:
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Setup MPICH
+      id:   setup1
+      uses: ./
+      with:
+        mpi: mpich
+    - run: test ${{ steps.setup1.outputs.mpi }} == mpich
```

@gerlero Sorry, yes, the my fault... The portable way to do it is the following, that is, replace `==` -> `=`.

```suggestion
    - run: test ${{ steps.setup1.outputs.mpi }} = mpich
```